### PR TITLE
6521 - Editor Paste not working when copied from Windows PDF Reader

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR will fix bug where paste function is not working on editor when copied from Windows Adobe Reader.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/6521

**Steps necessary to review your pull request (required)**:
- Use a Windows PC
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/editor/example-index.html
- Download the pdf in http://www.africau.edu/images/default/sample.pdf
- Open the PDF File
- Copy any line from the PDF File
- Paste it in the Editor

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@tjamesallen15 it seems like this breaks the ability to paste formatted word docs in the editor.
Try to make a word doc that has formatting like bold/fonts and copy paste and you can see its not working anymore.

- is there some way to detect the PDF text in the paste? 
- or figure out why its getting "stripped"?
- we did already have this option to paste as text http://localhost:4000/components/editor/test-paste-as-plain-text.html so what you added seems like the same thing as turning that on. last case we could tell them that they need this on?

**Comment by tjamesallen15** [comment]
> 

Hello @tmcconechy ,
When I try to copy from Acrobat Reader (Windows PDF Reader) all I get is text/html and text/rtf types, there are no text/html types which we need to get the bold/italics. See the image below.
copied from acrobat

I tried copying from MS Word and I get  the text/html fine. 
copied from word

So what I did is just check if the text/html is there. Now it is working with bold text copied from other sources.

**Review by tmcconechy — DISMISSED** [review]
Ok i think this will work. Thanks!

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,12 @@
 - `[Datagrid]` Fixed the overflowing of the multiselect dropdown on the page and pushes the container near the screen's edge. ([#6580](https://github.com/infor-design/enterprise/issues/6580))
 - `[Datagrid]` Fixed unselectRow on treegrid sending rowData incorrectly. ([#6548](https://github.com/infor-design/enterprise/issues/6548))
 - `[Datagrid]` Fixed incorrect rowData for grouping tooltip callback. ([NG#1298](https://github.com/infor-design/enterprise-ng/issues/1298))
+- `[Editor]` Fixed a bug where paste function is not working on editor when copied from Windows Adobe Reader. ([#6521](https://github.com/infor-design/enterprise/issues/6521))
```
Comment (ericangeles) [inline docs/CHANGELOG.md:20]: I see some duplicate logs. Can you fix it @tjamesallen15 ?

**Review by ericangeles — APPROVED** [review]

**Pull request merged**